Library/Dsqt/bridge/dsBridgeQuery.cpp

```diff
@@ -165,557 +173,640 @@ bool DsBridgeSqlQuery::tryLaunchBridgeSync()
+                " r.end_time,"            // 12
+                " r.effective_days"       // 13
+                " FROM record AS r"
+                " INNER JOIN lookup AS l ON l.uid = r.type_uid"
+                " WHERE r.complete = 1 AND r.visible = 1 AND (r.span_end_date IS NULL OR "
+                " date(r.span_end_date, '+5 day') > date('now'))"
+                " ORDER BY r.parent_slot ASC, r.rank ASC;";
+
+        timer.start();
+        recordQuery.exec(query);
+        qDebug() << "recordQuery took" << timer.elapsed() << "milliseconds";
+
+        query = "SELECT "
+                " lookup.uid,"    // 0
+                " lookup.app_key" // 1
+                " FROM lookup"
+                " WHERE lookup.type = 'select'";
+
+        timer.start();
+        selectQuery.exec(query);
+        qDebug() << "selectQuery took" << timer.elapsed() << "milliseconds";
+
+        query = "SELECT "
+                " record.uid,"                  // 0
+                " defaults.field_uid,"          // 1
+                " lookup.app_key,"              // 2
+                " defaults.field_type,"         // 3
+                " defaults.checked,"            // 4
+                " defaults.color,"              // 5
+                " defaults.text_value,"         // 6
+                " defaults.rich_text,"          // 7
+                " defaults.number,"             // 8
+                " defaults.number_min,"         // 9
+                " defaults.number_max,"         // 10
+                " defaults.option_type,"        // 11
+                " defaults.option_value,"       // 12
+                " lookup.app_key AS option_key" // 13
+                " FROM record"
+                " LEFT JOIN trait_map ON trait_map.type_uid = record.type_uid"
+                " LEFT JOIN lookup ON record.type_uid = lookup.parent_uid OR trait_map.trait_uid = "
+                "lookup.parent_uid "
+                " LEFT JOIN defaults ON lookup.uid = defaults.field_uid"
+                " WHERE EXISTS (select * from defaults where defaults.field_uid = lookup.uid);";
+
+        timer.start();
+        defaultsQuery.exec(query);
+        qDebug() << "defaultsQuery took" << timer.elapsed() << "milliseconds";
+
+        query = "SELECT "
+                " v.uid,"                   // 0
+                " v.field_uid,"             // 1
+                " v.record_uid,"            // 2
+                " v.field_type,"            // 3
+                " v.is_default,"            // 4
+                " v.is_empty,"              // 5
+                " v.checked,"               // 6
+                " v.color,"                 // 7
+                " v.text_value,"            // 8
+                " v.rich_text,"             // 9
+                " v.number,"                // 10
+                " v.number_min,"            // 11
+                " v.number_max,"            // 12
+                " v.datetime,"              // 13
```

Comment:
The schema schema-waffles-wlw is looking for v.date here, Which I'm pretty sure is correct. I changed the line to look for v.data and add v.time to the end of the query (spot 52). I, pretty sure this is the correct thing to do.